stories/card/card.stories.js

```diff
@@ -0,0 +1,719 @@
+import '../../dist/avatar.css';
+import '../../dist/badge.css';
+import '../../dist/button.css';
+import '../../dist/card.css';
+
+export default {
+    title: 'Components/Card',
+    parameters: {
+        description: `A Card is an app content container that represents a task or a topic on an overview page. 
+It is a smart component that uses UI annotations to render its content. Cards can show application content from different sources side by side – without requiring the user to switch screens.
+Cards can display different content:
+
+- a chart
+- a list
+- a table
+- informative text
+- a combination of two elements
+
+Card fields are not editable. A card can focus on a single object or topic, or on a group of objects. Cards sizes vary depanding on the layout.
+Several cards can reference the same applicaton but the information should not be repeated. 
+`,
+        tags: ['f3', 'a11y', 'theme', 'development']
+    }
+};
+
+
+export const cardAnatomy = () => `
+<div style="display:flex; justify-content:space-around; flex-wrap: wrap">
+    <div style="width: 300px; height: 400px; margin: 1rem;">
+        <div class="fd-card" role="region" aria-label="card-anatomy-1">
+            <a class="fd-card__header" tabindex="0" role="heading" aria-level="3" aria-roledescription="Card Heading" aaria-label="card-header">
```

Comment:
```suggestion
            <a class="fd-card__header" tabindex="0" role="heading" aria-level="3" aria-roledescription="Card Heading" aria-label="card-header">
```

Reply:
@prsdthkr Thanks for noticing this :) My keyboard is going crazy. The `e` key gives me at least `eee` for a single `e` :D I will fix it. Thanks again